app/models/declaration/ExportItem.scala

```diff
@@ -35,9 +37,15 @@ case class ExportItem(
   def hasFiscalReferences: Boolean =
     fiscalInformation.exists(_.onwardSupplyRelief == FiscalInformation.AllowedFiscalInformationAnswers.yes)
 
-  def isCompleted: Boolean =
-    procedureCodes.isDefined && isFiscalInformationCompleted && itemType.isDefined &&
-      packageInformation.nonEmpty && commodityMeasure.isDefined && additionalInformation.isDefined
+  val isCompleted: PartialFunction[DeclarationType, Boolean] = {
+    case DeclarationType.STANDARD | DeclarationType.SUPPLEMENTARY =>
+      procedureCodes.isDefined && isFiscalInformationCompleted && itemType.isDefined &&
+        packageInformation.nonEmpty && commodityMeasure.isDefined && additionalInformation.isDefined
+
+    case DeclarationType.SIMPLIFIED =>
+      procedureCodes.isDefined && isFiscalInformationCompleted && itemType.isDefined &&
+        packageInformation.nonEmpty && additionalInformation.isDefined
```

Comment:
If we extract validation to method like `supplementaryDecValidation` or some kind `baseValidation` we can reuse it for Standard and Supplementary like `baseValidation && commodityMeasure.isDefined`.

Non blocking comment, I'm not sure about naming here and the value.

Reply:
Generally i would not share validation rules here between journeys  - even if they looks similar